src/Apps/Consign/Routes/ConsignmentInquiry/ConsignmentInquiry.tsx

```diff
@@ -102,16 +120,34 @@ export const ConsignmentInquiry: React.FC<ConsignmentInquiryProps> = ({
         },
       })
 
+      console.log(
+        "[LOGD] input,  response, match, recipientEmail, !!recipientEmail= ",
+        input,
+        response,
+        match,
+        recipientEmail,
+        !!recipientEmail ? true : false
+      )
+
       const consignmentInquiryId =
         response.createConsignmentInquiry?.consignmentInquiryOrError
           ?.consignmentInquiry?.internalID
 
+      console.log("[LOGD] consignmentInquiryId = ", consignmentInquiryId)
+
       if (consignmentInquiryId) {
         trackEvent(tracks.sentConsignmentInquiry(consignmentInquiryId))
         router.replace({ pathname: "/sell" })
-        router.push("/sell/inquiry/sent")
+
+        router.push(
+          !!recipientEmail
+            ? `/sell/inquiry/${recipientEmail}/sent`
+            : "/sell/inquiry/sent"
+        )
       }
     } catch (error) {
+      console.log("[LOGD] error = ", error)
```

Comment:
👀

Reply:
Omg i'm sorry for all these console logs
was fighting with the router and used them a lot